test/suites/storage.sh

```diff
@@ -1,4 +1,9 @@
 test_storage() {
+  test_storage_common
+  test_storage_volume_snapshots
```

Comment:
Since we don't seem to be sharing any code here, can you just make this storage_snapshot.sh and have it treated as a separate test suite?

Granularity is good since we can jump between suites in the Jenkins UI.